src/sumoLogger.js

```diff
@@ -130,7 +135,7 @@ SumoLogger.prototype.log = function(msg, optConfig) {
   var testEl = isArray ? msg[0] : msg;
   var type = typeof testEl;
 
-  if (type === 'undefined' || (type === 'string' && testEl === '')) {
```

Comment:
Empty strings are already caught on line 124 (`if (!msg)`) so this check will never be reached